tests/transformers/test_scale.py

```diff
@@ -0,0 +1,134 @@
+from pandas import DataFrame
+from sklearn.preprocessing import MinMaxScaler as SklearnMinMaxScaler
+from sklearn.preprocessing import Normalizer as SklearnNormalizer
+from sklearn.preprocessing import StandardScaler as SklearnStandardScaler
+
+from pyreal.transformers.scale import MinMaxScaler, Normalizer, StandardScaler
+
+# MinMaxScaler tests
+data = [[2, 1, 3, 10], [4, 3, 4, 0], [6, 7, 2, 2]]
+columns = ["A", "B", "C", "D"]
+
+# test whether pyreal and sklearn scalers behave the same
+
+
+def test_fit_minmax():
+    skMinMax = SklearnMinMaxScaler()
+    skMinMax.fit(data)
+
+    pdData = DataFrame(data)
+    minmax = MinMaxScaler()
+    minmax.fit(pdData)
+
+    # our scaler objects are different types; test the attributes
+    assert all(
```

Comment:
Once you remove the unused member variables, you may not be able to do these checks anymore, in which case feel free to remove them and just check for errors on fit (or check the member variables of the wrapped transformer, like `minmax.wrapped_transformer.min_`)